vuecomponents/tool/pom.xml

```diff
@@ -0,0 +1,134 @@
+<?xml version="1.0"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>org.sakaiproject</groupId>
+        <artifactId>vuecomponents-parent</artifactId>
+        <version>23-SNAPSHOT</version>
+    </parent>
+
+    <name>Sakai Vue Components Tool</name>
+    <groupId>org.sakaiproject</groupId>
+    <artifactId>vuecomponents</artifactId>
+    <packaging>war</packaging>
+
+    <properties>
+        <frontend-maven-plugin.version>1.12.1</frontend-maven-plugin.version>
+        <frontend-maven-plugin.npmVersion>8.6.0</frontend-maven-plugin.npmVersion>
+        <frontend-maven-plugin.nodeVersion>v16.14.2</frontend-maven-plugin.nodeVersion>
+    </properties>
+
+    <profiles>
+        <profile>
+            <id>default</id>
+            <activation>
+                <activeByDefault>true</activeByDefault>
+            </activation>
+            <properties>
+                <sakai.skip.test.phase>test</sakai.skip.test.phase>
+            </properties>
+        </profile>
+        <profile>
+            <activation>
+                <property>
+                    <name>maven.test.skip</name>
+                </property>
+            </activation>
+            <id>skip-tests</id>
+            <properties>
+                <sakai.skip.test.phase>none</sakai.skip.test.phase>
+            </properties>
+        </profile>
```

Comment:
are these profiles needed?

Reply:
Yes. I did not find another way to disable the test runs when running maven with the -Dsakai.skip.test=true flag